commit c8b10b4512c5b4fe080f5eca97b7290cbd13e439
Author: Romaric Jodin <rjodin@google.com>
Date:   Thu Oct 9 14:53:08 2025 +0200

    meson: add vk_enum_defines.h to idep_vulkan_util_headers
    
    Adding vk_enum_defines.h to idep_vulkan_util_headers to help
    ninja-to-soong generate correct rules for the Android build system.
    
    Without it, ninja-to-soong is not able to figure out that this file is
    needed by targets depending on idep_vulkan_util_headers, leading to
    build errors with the file missing.
    
    Ref #14072
    
    Reviewed-by: Faith Ekstrand <faith.ekstrand@collabora.com>
    Part-of: <https://gitlab.freedesktop.org/mesa/mesa/-/merge_requests/37789>

diff --git a/src/vulkan/util/meson.build b/src/vulkan/util/meson.build
index 4915fc80f30..38f8a81d205 100644
--- a/src/vulkan/util/meson.build
+++ b/src/vulkan/util/meson.build
@@ -133,7 +133,7 @@ libvulkan_util = static_library(
 )
 
 idep_vulkan_util_headers = declare_dependency(
-  sources : [vk_dispatch_table[1], vk_enum_to_str[1],
+  sources : [vk_dispatch_table[1], vk_enum_to_str[1], vk_enum_to_str[2],
              vk_struct_type_cast[0], vk_extensions[1]],
   include_directories : include_directories('.')
 )
